### DISCUSSION TITLE
Is there any consequence of using 'fixPar' when interpreting results?

### DISCUSSION THREAD

**Discussion by Dekike** [bmcclintock/momentuHMM#61] [Q&A] [answered]
Hi Brett,

Thanks for creating the package `momentuHMM` and all the time you spend replying users' doubts.

I am assessing the effect of different environmental co-variables in the activity behaviour of a marine fish species using acoustic telemetry. I am running a HMM by individual using `fitHMM()` since I have only a few individuals and I think that the patterns can be different among them. So my data stream is the average activity of my fish at 15-min time bins (`meanAcc`). I am just defining two states (`Active` and `Resting`) after observing the histograms for `meanAcc` for all the individuals (two clear frequency peaks in `meanAcc`).

My doubt comes when I plot the histogram of the activity values and the estimated state-dependent probability distributions using `plot()` in the same way you do it in the provided examples. If I do not use `fixPar()`, I get estimated state-dependent probability distributions that do not seem proper to me (at least visually). I have tried different initial parameters but the state-dependent probability distributions end always in the same fit independently of the initial values. However, when I use `fixPar` I really get good state-dependent probability distributions. 

This is the fit when I do not use `fixPar` (independently of the initial parameters):
![Plot1](https://user-images.githubusercontent.com/65897655/101821063-bb798a00-3b27-11eb-9a66-6f604bee3fdc.png)
This is the fit when I do use `fixPar`:
![Plot2](https://user-images.githubusercontent.com/65897655/101821194-eb289200-3b27-11eb-9a5f-b0d41511f2da.png)

Thus, my question is: is it a problem to do not leave to the algorithm to estimate the parameters for the state-dependent probability distributions? I wonder if in this case, for example, it would be advisable to use `fixPar` or not.

Thanks for your time Brett.

**Comment by bmcclintock** [comment, marked as answer]
I'm not sure I follow you here, but it sounds like you want to use `fixPar` to fix the `meanAcc` probability distribution parameters to values that yield a desired result. This is not valid from a statistical standpoint. Assuming the freely estimated model is converging to the global maximum, the data are suggesting that the state-dependent distributions are perhaps not as distinct as you might have hoped and/or the lack of fit is attributable to some additional structure in the data that is not well explained by the model. Pseudo-residuals can be helpful for diagnosing lack of fit (see the `plotPR` function).

These are complex models, and there is a lot that needs to be understood in order to draw reliable inferences from them. I'd suggest carefully going through the [moveHMM](https://cran.r-project.org/package=moveHMM) vignettes, [momentuHMM](https://cran.r-project.org/package=momentuHMM) vignette, this recent [paper](https://doi.org/10.1111/ele.13610) and supplemental [tutorial](https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1111%2Fele.13610&file=ele13610-sup-0002-Supinfo.pdf), and other more general resources in the statistical literature (e.g. [Zucchini et al. 2016](https://www.routledge.com/Hidden-Markov-Models-for-Time-Series-An-Introduction-Using-R-Second-Edition/Zucchini-MacDonald-Langrock/p/book/9781482253832)) before proceeding further. If you don't have enough free time to invest in gaining a better understanding of these fundamentals, it might be worth considering collaborating with a statistician.

**Comment by Dekike** [comment]
Hi Brett,

I completely understood. I won't use `fixPart`. I will try reducing data since there is substantial autocorrelation for some individuals and a colleague said to me that one option to reduce autocorrelation is to remove part of the data (e.g, 1 out of 4 time-bins). I wonder if that could help to a better fit now that you talked about Pseudo-residuals.

Thanks for your time.
